14:37:29,840 ERROR main AllureResultsUtils:writeTestSuiteResult:151 - Error while marshaling testSuite
ru.yandex.qatools.allure.exceptions.AllureException: Can't create marshaller for class class ru.yandex.qatools.allure.model.TestSuiteResult
	at ru.yandex.qatools.allure.utils.AllureResultsUtils.createMarshallerForClass(AllureResultsUtils.java:183)
	at ru.yandex.qatools.allure.utils.AllureResultsUtils.marshaller(AllureResultsUtils.java:166)
	at ru.yandex.qatools.allure.utils.AllureResultsUtils.writeTestSuiteResult(AllureResultsUtils.java:146)
	at ru.yandex.qatools.allure.utils.AllureResultsUtils.writeTestSuiteResult(AllureResultsUtils.java:134)
	at ru.yandex.qatools.allure.Allure.fire(Allure.java:190)
	at ru.yandex.qatools.allure.testng.AllureTestListener.onFinish(AllureTestListener.java:82)
	at org.testng.TestRunner.fireEvent(TestRunner.java:1187)
	at org.testng.TestRunner.afterRun(TestRunner.java:976)
	at org.testng.TestRunner.run(TestRunner.java:606)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
Caused by: javax.xml.bind.JAXBException
 - with linked exception:
[java.lang.ClassNotFoundException: com.sun.xml.internal.bind.v2.ContextFactory]
	at javax.xml.bind.ContextFinder.newInstance(ContextFinder.java:241)
	at javax.xml.bind.ContextFinder.find(ContextFinder.java:477)
	at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:656)
	at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:599)
	at ru.yandex.qatools.allure.utils.AllureResultsUtils.createMarshallerForClass(AllureResultsUtils.java:181)
	... 20 more
Caused by: java.lang.ClassNotFoundException: com.sun.xml.internal.bind.v2.ContextFactory
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:582)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:185)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:496)
	at javax.xml.bind.ContextFinder.safeLoadClass(ContextFinder.java:594)
	at javax.xml.bind.ContextFinder.newInstance(ContextFinder.java:239)
	... 24 more
15:26:24,505 ERROR main AllureResultsUtils:writeTestSuiteResult:151 - Error while marshaling testSuite
ru.yandex.qatools.allure.exceptions.AllureException: Can't create marshaller for class class ru.yandex.qatools.allure.model.TestSuiteResult
	at ru.yandex.qatools.allure.utils.AllureResultsUtils.createMarshallerForClass(AllureResultsUtils.java:183)
	at ru.yandex.qatools.allure.utils.AllureResultsUtils.marshaller(AllureResultsUtils.java:166)
	at ru.yandex.qatools.allure.utils.AllureResultsUtils.writeTestSuiteResult(AllureResultsUtils.java:146)
	at ru.yandex.qatools.allure.utils.AllureResultsUtils.writeTestSuiteResult(AllureResultsUtils.java:134)
	at ru.yandex.qatools.allure.Allure.fire(Allure.java:190)
	at ru.yandex.qatools.allure.testng.AllureTestListener.onFinish(AllureTestListener.java:82)
	at org.testng.TestRunner.fireEvent(TestRunner.java:1187)
	at org.testng.TestRunner.afterRun(TestRunner.java:976)
	at org.testng.TestRunner.run(TestRunner.java:606)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
Caused by: javax.xml.bind.JAXBException
 - with linked exception:
[java.lang.ClassNotFoundException: com.sun.xml.internal.bind.v2.ContextFactory]
	at javax.xml.bind.ContextFinder.newInstance(ContextFinder.java:241)
	at javax.xml.bind.ContextFinder.find(ContextFinder.java:477)
	at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:656)
	at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:599)
	at ru.yandex.qatools.allure.utils.AllureResultsUtils.createMarshallerForClass(AllureResultsUtils.java:181)
	... 20 more
Caused by: java.lang.ClassNotFoundException: com.sun.xml.internal.bind.v2.ContextFactory
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:582)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:185)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:496)
	at javax.xml.bind.ContextFinder.safeLoadClass(ContextFinder.java:594)
	at javax.xml.bind.ContextFinder.newInstance(ContextFinder.java:239)
	... 24 more
